src/Resources/contao/dca/tl_user.php

```diff
@@ -28,4 +25,22 @@
 	'sql'                     => "varchar(64) NOT NULL default ''"
 );
 
-
+class tl_user extends Backend
+{
+    public function onloadCallback(): void
```

Comment:
Ich bin im Thema "DCA-Manipulation" nicht so drin, deshalb kann ich es nicht richtig beurteilen. Warum wird hier mit einem onload_callback gearbeitet, um die Paletten zu manipulieren? Z.B. in src/Resources/contao/dca/tl_page.php (selber PR) wird der PaletteManipulator meines Erachtens ohne onload_callback benutzt. In tl_content.php (selber PR) wird der Callback aber auch benutzt.